pwned_passwords_django/api.py

```diff
@@ -13,10 +15,19 @@ def pwned_password(password):
     """
     password_hash = hashlib.sha1(password.encode('utf-8')).hexdigest().upper()
     prefix, suffix = password_hash[:5], password_hash[5:]
-    results = [
-        int(line.split(':')[1]) for line in
-        requests.get(API_ENDPOINT.format(prefix)).text.splitlines()
-        if line.startswith(suffix)
-    ]
+    try:
+        results = [
+            int(line.partition(':')[2])
+            for line in requests.get(
+                url=API_ENDPOINT.format(prefix),
+                timeout=0.6,
```

Comment:
The `0.6` ends up hard-coded in a lot of places. Could you move it to a constant in `api.py`, maybe call it `REQUEST_TIMEOUT`?